onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -382,7 +353,10 @@ static bool ParseSessionConfigs(const std::string& configs_string,
         test_config.run_config.intra_op_thread_affinities = ToUTF8String(optarg);
         break;
       case 'C': {
-        if (!ParseSessionConfigs(ToUTF8String(optarg), test_config.run_config.session_config_entries)) {
+        ORT_TRY {
+          ParseSessionConfigs(ToUTF8String(optarg), test_config.run_config.session_config_entries);
+        }
+        ORT_CATCH(...) {
```

Comment:
nit: does this swallow the exception and provide no error output to the user? 

avoid '...' unless really necessary (most likely isn't here) and only after catching std::exception so that you can output the error info.

Reply:
Sure.